pom.xml

```diff
@@ -340,7 +341,7 @@
         <dependency>
             <groupId>org.hamcrest</groupId>
             <artifactId>hamcrest-core</artifactId>
-            <version>2.2</version>
+            <!--<version>2.2</version>-->
             <scope>test</scope>
         </dependency>
```

Comment:
nit: I think this dependency already comes in transitively from `jenkins-test-harness` at this version. Can it be deleted?

Reply:
Yea, I just forgot to remove it after all my back and forth testing what I need to do to make the require upper bounds to be gone :)